src/site/stages/build/drupal/graphql/homePage.graphql.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Home page
+ */
+
+const menu = 'homepage-top-tasks-blocks';
+const hubListQueue = 'home_page_hub_list';
+const promoBlocksQueue = 'home_page_promos';
+
+const {
+  featureFlags,
+  enabledFeatureFlags,
+} = require('../../../../utilities/featureFlags');
```

Comment:
Heads up: I'm working on a PR to have these flags get pulled down from Drupal, so this will be out of date.

If you merge yours first, that won't matter, but I figured I should give you a heads up. :smile:

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/10586